### PR TITLE
refactor(tutor-mode): Added CLI subdirectories

### LINKED ISSUE #1601: CLI tutor mode: Support subdirectories for "import" commands
Per @lidel in https://github.com/ipfs-shipyard/ipfs-webui/pull/1572#pullrequestreview-475583140:

> CLI tutor for Import commands ignores the fact we may be in a subdirectory.
> Ideally `/<dest-name>` in import commands would be replaced with `"${path}/<dest-name>"` where `path` is the current dir in MFS. I started doing that in [`3a337de`](https://github.com/ipfs-shipyard/ipfs-webui/commit/3a337de195cdecf9a9e41e4a3486e511b736897b) but had no bandwidth to wire it up.
> If anyone has time to ensure `path` is set for [these actions](https://github.com/jay-dee7/ipfs-webui/blob/jay-dee7/cli-tutor-mode/src/files/modals/Modals.js#L157-L160) that would be even better for user experience.

To do: Complete the work referenced above, and if appropriate add/rework any CLI tutor mode functionality as discovered post-2.11 launch.

### PR DESCRIPTION
**Pull request by melegiul**
### Summary:
- Reads modals root prop to insert current MFS path in CLI command of tutor mode
- Applied for all import commands at file menu

Closes #1601

### Screenshot:

mfs-subdirectory

### CODE REVIEW

**Comment by jessicaschilling** [comment]
Built locally and did a quick functional review and LGTM 😊  but I want to defer to @lidel for a more thorough examination. @lidel, can you please take a look when you get a chance? Thank you!

**Review by jessicaschilling — APPROVED** [review]
Thanks so much for PRing this, @melegiul! LGTM but want to wait to merge until we get approval from @lidel.

**Comment by melegiul** [comment]
Thank you @jessicaschilling for your review :smile: I will wait for the  pending approval.

**Review by lidel — APPROVED** [review]
Really happy about attention to detail like that :heart: 
Thank you @melegiul

**Comment by jessicaschilling** [comment]
PS: @melegiul, any other contributions you'd like to make would be more than welcome! 😊

**Comment by melegiul** [comment]
Thank you for your prompt feedback @lidel , @jessicaschilling I would be very happy to contribute another one soon :smile:

**Pull request merged**
